roles/pulp-webserver/templates/nginx.conf.j2

```diff
@@ -81,9 +81,5 @@ http {
 
         include conf.d/*;
 
-        location / {
```

Comment:
The removal of this rule will break static media for projects outside of galaxy_ng. To fix, can a `/static/` route match to pulp-api be added here to replace `/`. That would be driven by the role variable that allows users to configure the static url.

Reply:
sounds good, in this case I will have to update the galaxy snippets otherwise it would break due duplication of `/static/` route